pykesko/src/lib.rs

```diff
@@ -169,6 +189,15 @@ impl KeskoApp {
         }
     }
 
+    pub fn publish_flatbuffers(&mut self, flatbuffer: Vec<u8>) {
```

Comment:
What does this do?

Reply:
This publishes a binary message (encoded with FlatBuffers) in the Kesko (Bevy) app. This is useful because our plugins can subscribe to all of these messages, and handle them if the format matches one of their actions. This allows us to avoid extending the Kesko command list. Instead, we can describe the possible commands per plugin. Like here: kesko/crates/placeholder_box/definitions/clear.fbs

Reply:
Thank you, it all makes more sense now